src/api/firebase.js

```
@@ -219,10 +220,17 @@ export async function updateItem(listPath, itemId, { totalPurchases }) {
 	}
 }
 
-export async function deleteItem() {
+export async function deleteItem(listPath, itemId) {
 	/**
 	 * TODO: Fill this out so that it uses the correct Firestore function
 	 * to delete an existing item. You'll need to figure out what arguments
 	 * this function must accept!
 	 */
+	try {
+		const itemRef = doc(db, listPath, 'items', itemId);
+		await deleteDoc(itemRef);
+		return 'Item has been deleted successfully';
+	} catch (error) {
+		console.log(error);
+	}
```

you can throw the error here and catch it in the component where you are calling deleteItem for appropriate handling